tests/cli/test_workflow.py

```diff
@@ -1253,7 +1252,7 @@ def test_workflow_cycle_detection(run_shell, project, capsys, client):
     assert b"Cycles detected in execution graph" in result[0]
 
 
-@pytest.mark.skipif(sys.platform == "darwin", reason="GitHub macOS image doesn't include Docker")
+# @pytest.mark.skipif(sys.platform == "darwin", reason="GitHub macOS image doesn't include Docker")
```

Comment:
I'm guessing this didn't work, looking at the failed test?

Reply:
Oh my bad I did that so that it runs on my computer. I meant to put it back the way it was.